plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouter.sol

```diff
@@ -28,9 +27,8 @@ contract PaymentStandardExitRouter is
     using PaymentChallengeStandardExit for PaymentChallengeStandardExit.Controller;
     using PaymentProcessStandardExit for PaymentProcessStandardExit.Controller;
     using BondSize for BondSize.Params;
-    using ExitBounty for ExitBounty.Params;
 
-    // Initial bond size = 70000 (gas cost of challenge) * 20 gwei (current fast gas price) * 10 (safety margin)
+    // Initial bond size = 70000 (gas cost of challenge) * 40 gwei (current fast gas price) * 5 (safety margin)
     uint128 public constant INITIAL_BOND_SIZE = 14000000000000000 wei;
```

Comment:
What initial values should we have for SE and IFE bond and bounty?

Reply:
Did you update the comment but not the value?

Reply:
Should we chose a single gas price across initial bond values?